zgw/taak/src/main/kotlin/nl/nlportal/zgw/taak/domain/TaakFormulierV2.kt

```diff
@@ -3,4 +3,14 @@ package nl.nlportal.zgw.taak.domain
 data class TaakFormulierV2(
     val soort: String,
     val value: String,
-)
\ No newline at end of file
+) {
+    companion object {
+        fun migrate(taakFormulier: TaakFormulier): TaakFormulierV2 {
+            val value = taakFormulier.value
+            if (value.startsWith("http")) {
+                TaakFormulierV2("url", value)
```

Comment:
Hier gebeurt nu niks meer mee is dat de bedoeling?

Reply:
Was een return vergeten, heb het aangepast